bin/src/hickory-dns.rs

```diff
@@ -473,13 +469,11 @@ fn run() -> Result<(), String> {
 
     if !args.disable_udp && !config.get_disable_udp() {
         // load all udp listeners
-        for udp_socket in &sockaddrs {
-            info!("binding UDP to {:?}", udp_socket);
+        for addr in &listen_addrs {
+            info!("binding UDP to {addr:?}");
             let udp_socket = runtime
-                .block_on(UdpSocket::bind(udp_socket))
```

Comment:
If we're going to call a sync function, can we just get rid of the `runtime.block_on()` wrappers?